qancode/tasks.py

```diff
@@ -475,7 +475,7 @@ def take_screenshot(self):
         scroll_height = self.driver.execute_script(
             'return document.body.scrollHeight;')
         if ((self.driver.capabilities['browserName'] == 'safari')
-                or (client_height == scroll_height)):
+                or (np.allclose(client_height, scroll_height, rtol=0.0025))):
```

Comment:
constants like RTOL should be put at the top of the file.

Reply:
Thanks, defined at top of class since nothing else is going to use it. Okay?